client.go

```diff
@@ -83,57 +101,105 @@ func New() *NebulaPool {
 	}
 }
 
+// NewSSLConfig return sslConfig
+func (np *NebulaPool) NewSSLConfig(rootCAPath, certPath, privateKeyPath string) {
+	np.sslconfig = &sslConfig{
+		rootCAPath:     rootCAPath,
+		certPath:       certPath,
+		privateKeyPath: privateKeyPath,
+	}
```

Comment:
For SelfSign, is password in privateKeyPath file?

Reply:
nebula-go doesn't support password yet ..